package-lock.json

```diff
@@ -5146,7 +5146,8 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true
+          "bundled": true,
+          "optional": true
```

Comment:
These `optional` keys are new. How do we feel about them being in `package-lock.json`? I can't say I've seen them before.

@RichAyotte @knicklabs @nbering

Reply:
Good question. I don't have an opinion, but could look into what it means if you want.

Reply:
@adam12 @nbering It looks like dependencies changed in version `3.13.1` of `js-yaml` so a bunch of packages are no longer required. This was not an ideal fix. Closing this out. I'll submit a PR to fix this vulnerability.